azure-pipelines.yml

```diff
@@ -3,7 +3,7 @@ trigger:
 
 variables:
   # Change the following along with adding new TEST_START_INDEX.
-  TEST_COUNT: 18
+  TEST_COUNT: 16
```

Comment:
At some point we should just write a script that generates these Azure yml files as having to keep twiddling the start indexes/count is not ideal.